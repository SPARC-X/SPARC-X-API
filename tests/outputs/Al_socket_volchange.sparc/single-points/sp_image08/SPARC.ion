# Ion File Generated by SPARC ASE Calculator
# 
# ASE-SORT:
# 0 1 2 3
# END ASE-SORT

ATOM_TYPE: Al
N_TYPE_ATOM: 4
PSEUDO_POT: 13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
COORD:
0.01671813777221 0.20003587269869 -0.35688634394130
-0.25412263026917 3.29093975049837 4.15362544634555
4.02861284612154 0.40410273785996 3.85758917856974
3.89087588287684 3.49496828243622 0.35127603099891

