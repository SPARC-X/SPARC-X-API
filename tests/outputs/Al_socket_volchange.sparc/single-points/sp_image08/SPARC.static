***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Al:
      0.0022519683       0.0269452527       0.9519266590
      0.9657691474       0.4432965043       0.5595020816
      0.5426626214       0.0544334885       0.5196253738
      0.5241091629       0.4707795766       0.0473176200
Total free energy (Ha): -8.984190762502326E+00
Atomic forces (Ha/Bohr):
  1.3917785591E-02  -4.4169443843E-02   3.8341549666E-02
  4.3295731608E-02   6.6870036035E-02  -2.6412225841E-02
 -4.4462499659E-02  -4.9668713085E-02  -9.5925769433E-03
 -1.2751017540E-02   2.6968120893E-02  -2.3367468818E-03
Stress (GPa): 
  3.7152930067E+00  -5.3592154346E+00  -1.9862739090E+00 
 -5.3592154346E+00  -4.5853884151E+01   7.4079969226E+00 
 -1.9862739090E+00   7.4079969226E+00  -4.9712135261E+01
