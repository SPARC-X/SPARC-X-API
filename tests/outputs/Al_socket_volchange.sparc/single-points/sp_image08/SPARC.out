***************************************************************************
*                       SPARC (version Oct 31, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Thu Jan 18 08:03:32 2024                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 1 1 1 
LATVEC:
7.423789085350370 0.000000000000000 0.000000000000000 
0.000000000000000 7.423789085350370 0.000000000000000 
0.000000000000000 0.000000000000000 7.423789085350370 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.000950043469
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 12
CHEB_DEGREE: 17
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-03
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-05
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.45E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0
MIXING_PARAMETER: 1
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
NUM_CHEFSI: 1
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: SPARC
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
7.423789085350370 0.000000000000000 0.000000000000000 
0.000000000000000 7.423789085350370 0.000000000000000 
0.000000000000000 0.000000000000000 7.423789085350370 
Volume: 4.0914464784E+02 (Bohr^3)
Density: 2.6378483641E-01 (amu/Bohr^3), 2.9559393927E+00 (g/cc)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 2
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  2
Mesh spacing                       :  0.494919 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  SPARC.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
Atomic mass                        :  26.9815385
Pseudocharge radii of atom type 1  :  5.94 5.94 5.94 (x, y, z dir)
Number of atoms of type 1          :  4
Estimated total memory usage       :  3.01 MB
Estimated memory per processor     :  1.50 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2534533498E+00        2.622E-01        0.020
2            -2.2523835411E+00        2.637E-01        0.007
3            -2.2466241400E+00        1.290E-01        0.007
4            -2.2465925076E+00        1.071E-01        0.007
5            -2.2462017423E+00        7.487E-02        0.006
6            -2.2460911280E+00        4.311E-02        0.007
7            -2.2460549603E+00        1.264E-02        0.006
8            -2.2460616288E+00        2.693E-02        0.006
9            -2.2460469230E+00        3.802E-03        0.006
10           -2.2460474109E+00        4.674E-03        0.006
11           -2.2460476906E+00        3.827E-04        0.006
Total number of SCF: 11    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2460476906E+00 (Ha/atom)
Total free energy                  : -8.9841907625E+00 (Ha)
Band structure energy              :  7.0145849803E-01 (Ha)
Exchange correlation energy        : -4.5057794622E+00 (Ha)
Self and correction energy         : -1.2498677095E+01 (Ha)
-Entropy*kb*T                      : -2.7911635035E-03 (Ha)
Fermi level                        :  1.4804362118E-01 (Ha)
RMS force                          :  6.0330168383E-02 (Ha/Bohr)
Maximum force                      :  8.3926919214E-02 (Ha/Bohr)
Time for force calculation         :  0.012 (sec)
Pressure                           :  3.0616908802E+01 (GPa)
Maximum stress                     :  4.9712135261E+01 (GPa)
Time for stress calculation        :  0.021 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.208 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
